e2e/tests/components/toolbar-search/toolbar-search.e2e-spec.ts

```diff
@@ -31,6 +31,14 @@ describe('Toolbar Search', () => {
 
     });
 
+    it('should display the name of the selected number', async () => {
```

Comment:
I don't think this name makes it clear what the test is doing. Is it just checking that 'One' is selected by default?